src/jwplayer/utils/useJwpAdsSetupComplete.ts

```diff
@@ -21,17 +23,19 @@ export default function useJwpAdsSetupComplete(): JwpAdsSetupCompleteResult {
 	const [jwpAdsSetupComplete, setJwpAdsSetupComplete] = useState(null);
 	const [vastUrl, setVastUrl] = useState(null);
 	const [strategyRulesEnabled, setStrategyRulesEnabled] = useState(false);
+	const [vastXml, setVastXml] = useState('');
```

Comment:
Same here. IMO vastXml could be undefined by default.
`const [vastXml, setVastXml] = useState<string>();`